content/src/components.ts

```diff
@@ -66,11 +67,19 @@ export async function initComponentsWithEnv(env: Environment): Promise<AppCompon
   const challengeSupervisor = new ChallengeSupervisor()
 
   const catalystFetcher = FetcherFactory.create({ env })
-  const daoClient = DAOClientFactory.create(env)
-  const authenticator = AuthenticatorFactory.create(env)
   const contentFolder = path.join(env.getConfig(EnvironmentConfig.STORAGE_ROOT_FOLDER), 'contents')
   const storage = await createFileSystemContentStorage({ fs }, contentFolder)
 
+  const ethNetwork: string = env.getConfig(EnvironmentConfig.ETH_NETWORK)
+  const ethereumProvider = new HTTPProvider(
+    `https://rpc.decentraland.org/${encodeURIComponent(ethNetwork)}?project=catalyst-content`
+  )
```

Comment:
@pedrotambo FYI

Reply:
ohh okay, so i'll let you finish your crusade in peace 😋 . Nice work!